validators/performance/aiperf_entrypoint.py

```
@@ -0,0 +1,108 @@
+    return value
+
+
+def main() -> int:
+    artifact_dir = _require_env("AICR_AIPERF_ARTIFACT_DIR")
+    sentinel = _require_env("AICR_RESULT_SENTINEL")
+
+    cmd = [AIPERF_BIN, "profile", *sys.argv[1:]]
+
+    # No shell: argv is passed straight to execvp, so no element is re-parsed.
+    # stdout/stderr are inherited so benchmark progress reaches the pod log.
+    completed = subprocess.run(cmd, check=False)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)